pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -49,9 +49,8 @@ def __init__(
     ) -> None:
         self._accelerator = accelerator
```

Comment:
Why accelerator here is different?

Reply:
There wasn't a property for it here when I opened the PR. Let's fix it :)

```suggestion
        self.accelerator = accelerator
```